src/vs/editor/contrib/folding/browser/folding.css

```diff
@@ -6,40 +6,31 @@
 .monaco-editor .margin-view-overlays .folding {
 	margin-left: 5px;
 	cursor: pointer;
-}
-
-.monaco-editor .margin-view-overlays .folding::after {
 	background-repeat: no-repeat;
```

Comment:
why are these changes part of this PR? Is the change in the editor core forcing these changes?